docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -414,7 +414,13 @@ To specify limits on the number of ops/assets tagged with a particular concurren
 
 The concurrency key should match the name that the op/asset is tagged with. For example, if the op/asset is tagged with `dagster/concurrency_key: redshift`, then the concurrency key should be `redshift`.
 
----
+- With a global concurrency limit set, it is useful to configure your instance to automatically free slots for canceled / failed runs. Certain execution errors can cause steps to exit without freeing an occupied concurrency slot. To automatically recover from this state, you can configure your instance's run monitoring to automatically free concurrency slots for runs that have reached some terminal state (e.g. failed or canceled runs). You can configure the interval between a run end and the slot clean up in your `dagster.yaml`:
```

Comment:
What execution errors would lead to the state you mention here? Should we mention them?